python-sdk/tutorials/nuimages_tutorial.ipynb

```diff
@@ -196,7 +198,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "nuim.render_image(sd_token_camera, with_category=True, with_attributes=False, box_line_width=0, render_scale=2)"
+    "nuim.render_image(sd_token_camera, with_annotations='all',\n",
+    "                  with_category=True, with_attributes=False, box_line_width=0, render_scale=2)"
```

Comment:
also i noticed in the notebook, is there a way to set the transparency of the sparse/dense depth map?

Reply:
```
`list_cameras()` shows us how many camera entries 
```
camera entries should be changed to calibrated sensor or?

Reply:
add `sort_by` to let users know that feature exists:
```
nuim.list_attributes(sort_by='freq')
```

Reply:
> camera entries should be changed to calibrated sensor or?

Yes, let's change it to `list_sensors()`. The calibrated sensors are just one column here, there are also samples.